Library/Homebrew/extend/os/linux/utils/bottles.rb

```diff
@@ -0,0 +1,35 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  module Bottles
+    class << self
+      alias generic_tag tag
+      undef tag
+
+      def tag(symbol = nil)
+        return super(symbol) if symbol.present?
+
+        OS::LINUX_CI_OS_VERSION.downcase.delete(" .")
```

Comment:
This should return a `Tag` object.

This doesn't seem to take arch into account either.